<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxConnection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>210</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Параметры соединения</string>
    </property>
    <widget class="QLineEdit" name="edIP">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edPortIn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>2525</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelIP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>44</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPortIn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>74</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Соединить</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPortOut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>104</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Out</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edPortOut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>2424</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbControl">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Управление</string>
    </property>
    <widget class="QLineEdit" name="edHorizontalVectorX">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbHorizontalVectorX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HorizontalVectorX</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edHorizontalVectorY">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbHorizontalVectorY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HorizontalVectorY</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbAngularVelocityZ">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AngularVelocityZ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edAngularVelocityZ">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbVericalThrust">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>VericalThrust</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edVericalThrust">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbDepth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Depth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edDepth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbManipulatorState">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ManipulatorState</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edManipulatorState">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbManipulatorRotate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ManipulatorRotate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edManipulatorRotate">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbCameraRotate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CameraRotate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edCameraRotate">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edResetInitialization">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbResetInitialization">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ResetInitialization</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edLightsState">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbLightsState">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LightsState</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
